### PR TITLE
WT-6479 Don't insert updates after the first globally visible update to the history store

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

**Comment by quchenhao** [comment]
This fixes both WT-6479 and WT-6498. There is a single test failure in extended mongodb patch build https://spruce.mongodb.com/task/mongodb_mongo_v4.4_ubuntu1604_debug_replica_sets_large_txns_format_jscore_passthrough_patch_09eac8deb5d7d953dc974010c10bef577fa30dee_5f0169e861837d57b0cf2088_20_07_05_05_51_05/logs. The stress test is OK.

File: src/history/hs.c [hunk 1]
```diff
@@ -799,6 +815,8 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi)
             /* If we've seen a smaller timestamp before, use that instead. */
             if (min_insert_ts < upd->start_ts) {
                 /*
+                 * Only fix out of order timestamp if we haven't seen an update without timestamp.
```
Comment (agorrod) [inline src/history/hs.c:818]: The comment doesn't seem accurate. We still fix them up, we just don't reset them back to WT_TS_NONE (that's what it looks like the code is doing anyway, and what I expect it to do).

Comment (quchenhao) [reply src/history/hs.c]: Fixed. @agorrod Can I merge it?

**Comment by agorrod** [comment]
@quchenhao there have been changes here since the last round of stress testing - please complete another round of stress testing before merge.

**Comment by quchenhao** [comment]
Merge as stress tests are green

**Pull request merged**